src/browser_action/vitals.js

```diff
@@ -12,7 +12,7 @@
 */
 
 (async () => {
-  const src = chrome.runtime.getURL('node_modules/web-vitals/dist/web-vitals.es5.min.js');
+  const src = chrome.runtime.getURL('node_modules/web-vitals/dist/web-vitals.js');
```

Comment:
@addyosmani v2 of the library doesn't have this file so I think it's ok to replace it with regular old `web-vitals.js`. I may be missing some historical context behind why this particular file was used though.

Tested the extension and all functionality/options still appear to be WAI. Would you be able to test the branch from source as well to verify?

Reply:
I'll test this now and verify if all works correctly.